lib/Sema/CSGen.cpp

```diff
@@ -3129,6 +3134,19 @@ namespace {
       return tv;
     }
 
+    static bool isTriggerFallbackDiagnosticBuiltin(UnresolvedDotExpr *UDE,
+                                                   ASTContext &Context) {
+      auto *DRE = dyn_cast<DeclRefExpr>(UDE->getBase());
+      if (!DRE)
+        return false;
+
+      if (DRE->getDecl() != Context.TheBuiltinModule)
+        return false;
+
+      auto member = UDE->getName().getBaseIdentifier().str();
```

Comment:
`getBaseIdentifier().str()` will assert for special names. How about `getBaseName().userFacingName()`?

Reply:
Thanks!